deployment/cloud-run/secure_api_server.py

```
@@ -52,30 +57,37 @@ def home():  # Changed from api_root to home to avoid conflict with Flask-RESTX'
+        authorizations={
+            'apikey': {
+                'type': 'apiKey',
+                'in': 'header',
+                'name': 'X-API-Key'
+            }
+        },
+        security='apikey'
+    )
+    logger.info("✅ Flask-RESTX API initialized successfully")
+except Exception as e:
+    logger.error(f"❌ Flask-RESTX API initialization failed: {str(e)}")
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)